QIDITechnology/QIDITechnology.ini

```diff
@@ -0,0 +1,3300 @@
+inherits = *X-smart 3*; *0.40mm Fine*
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[print:0.48mm Fine @X-smart 3 0.8 nozzle]
+inherits = *X-smart 3*; *0.48mm Quick*
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+# Common filament preset
+[filament:*common*]
+advance_pressure = 0.031
+bed_temperature = 60
+bridge_fan_speed = 100
+compatible_printers = 
+compatible_printers_condition = 
+compatible_prints = 
+compatible_prints_condition = 
+cooling = 1
+disable_fan_first_layers = 1
+disable_rapid_cooling_fan_first_layers = 3
+enable_advance_pressure = 1
+enable_auxiliary_fan = 100
+enable_auxiliary_fan_unseal = 0
+enable_dynamic_fan_speeds = 0
+enable_volume_fan = 100
+end_filament_gcode = "; Filament-specific end gcode \n;END gcode for filament\n"
+extrusion_multiplier = 0.98
+fan_always_on = 1
+fan_below_layer_time = 60
+filament_colour = #3399FF
+filament_cooling_final_speed = 3.4
+filament_cooling_initial_speed = 2.2
+filament_cooling_moves = 4
+filament_cost = 0
+filament_density = 1.24
+filament_deretract_speed = nil
+filament_diameter = 1.75
+filament_load_time = 0
+filament_loading_speed = 28
+filament_loading_speed_start = 3
+filament_max_volumetric_speed = 20
+filament_minimal_purge_on_wipe_tower = 15
+filament_multitool_ramming = 0
+filament_multitool_ramming_flow = 10
+filament_multitool_ramming_volume = 10
+filament_notes = ""
+filament_purge_multiplier = 100
+filament_ramming_parameters = "120 100 6.6 6.8 7.2 7.6 7.9 8.2 8.7 9.4 9.9 10.0| 0.05 6.6 0.45 6.8 0.95 7.8 1.45 8.3 1.95 9.7 2.45 10 2.95 7.6 3.45 7.6 3.95 7.6 4.45 7.6 4.95 7.6"
+filament_retract_before_travel = nil
+filament_retract_before_wipe = nil
+filament_retract_layer_change = nil
+filament_retract_length = nil
+filament_retract_lift = nil
+filament_retract_lift_above = nil
+filament_retract_lift_below = nil
+filament_retract_restart_extra = nil
+filament_retract_speed = nil
+filament_settings_id = 
+filament_shrink = 100
+filament_soluble = 0
+filament_spool_weight = 0
+filament_stamping_distance = 0
+filament_stamping_loading_speed = 20
+filament_toolchange_delay = 0
+filament_travel_max_lift = nil
+filament_travel_ramping_lift = nil
+filament_travel_slope = nil
+filament_type = PLA
+filament_unload_time = 0
+filament_unloading_speed = 90
+filament_unloading_speed_start = 100
+filament_vendor = QIDI
+filament_wipe = nil
+filament_wipe_distance = nil
+first_layer_bed_temperature = 60
+first_layer_temperature = 210
+full_fan_speed_layer = 0
+idle_temperature = 100
+inherits = 
+max_fan_speed = 100
+min_fan_speed = 100
+min_print_speed = 20
+overhang_fan_speed_0 = 0
+overhang_fan_speed_1 = 0
+overhang_fan_speed_2 = 0
+overhang_fan_speed_3 = 0
+seal_print = 1
+slowdown_below_layer_time = 8
+smooth_time = 0.03
+start_gcode = "; Filament gcode\n{if (enable_advance_pressure[0] == true)}M900 K[advance_pressure]{else}M900 K0{endif} T[smooth_time]\n"
```

Comment:
This line is not correct.
`start_gcode` is not valid key within filament section. The correct key is `start_filament_gcode`.

Variables used in your filament start gcode does not exist in PrusaSlicer (likely taken from different slicer). Those variables must be removed, otherwise every slicing will end by error message.

Specific M900 K values should be added to the filament_start_gcode of each filament, instead of `advance_pressure`, which also does not exist in PrusaSlicer.